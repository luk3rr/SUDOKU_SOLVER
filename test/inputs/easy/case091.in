046010209 000020000 090070600 920300000 670000035 800000004 400902006 089005007 000700000 
