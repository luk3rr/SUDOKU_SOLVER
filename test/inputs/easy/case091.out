Solving the following grid:
0 4 6 | 0 1 0 | 2 0 9 
0 0 0 | 0 2 0 | 0 0 0 
0 9 0 | 0 7 0 | 6 0 0 
------+-------+------
9 2 0 | 3 0 0 | 0 0 0 
6 7 0 | 0 0 0 | 0 3 5 
8 0 0 | 0 0 0 | 0 0 4 
------+-------+------
4 0 0 | 9 0 2 | 0 0 6 
0 8 9 | 0 0 5 | 0 0 7 
0 0 0 | 7 0 0 | 0 0 0 


Solution found :')

5 4 6 | 8 1 3 | 2 7 9 
7 3 8 | 6 2 9 | 5 4 1 
1 9 2 | 5 7 4 | 6 8 3 
------+-------+------
9 2 4 | 3 5 7 | 1 6 8 
6 7 1 | 2 4 8 | 9 3 5 
8 5 3 | 1 9 6 | 7 2 4 
------+-------+------
4 1 7 | 9 3 2 | 8 5 6 
2 8 9 | 4 6 5 | 3 1 7 
3 6 5 | 7 8 1 | 4 9 2 

Algorithm: BFS
Total time: 178 ms
Total expanded states: 24888
